tests/testthat/test-uploadAttachment.R

```diff
@@ -0,0 +1,25 @@
+context("uploadAttachment")
+
+test_that("Correct HTTP request is made", {
+  with_mock(
+    `httr::request_perform` = function(req, handle, refresh) {
+      expect_equal(req$method, "POST")
+      expect_equal(req$url, "https://my.server.com/api/v1/experiments/591a3b441d725115208a6fdb/attachments")
+      # Not sure the best way to assert on this.
+      response = httptest::fake_response(
+        req$url,
+        req$method,
+        content='{"__v":0,"_id":"591a3b441d725115208a6fda","filename":"test.txt"}',
```

Comment:
That looks like maybe an old response. It should be like
`{"_id":"591a3b441d725115208a6fda","filename":"text.txt","experimentId":"591a3b441d725115208a6fdb","crc32c":"4d9f30b7","md5":"b6cb0f190337db50a672e9f3dc616e10","size":9}`